packages/app/templates/extensions/projects/flow_action/shopify.extension.toml.liquid

```diff
@@ -1,26 +1,25 @@
-name = "{{name}}"
+name = "{{ name }}"
+description = "Your description"
 type = "flow_action"
 
-[task]
-title = "extension-title"
-description = "extension-description"
-url = "https://runtime-endpoint.com"
-validation_url = "https://validation-url"
-custom_configuration_page_url = "https://custom-configuration-page-url"
-custom_configuration_page_preview_url = "https://custom-configuration-page-preview-url"
+[extension]
+type = "flow_action"
+handle = "{{ handle }}"
```

Comment:
where are you going to get the handle from?
this is not currently supported by the CLI 🤔

Reply:
right now the handle is just a slugified name from the extension name :D. I don't know where I'd write/fetch the handle name yet.